docs/hub/datasets-image.md

```diff
@@ -0,0 +1,123 @@
+# Image Dataset
+
+This guide will show you how to configure your dataset repository with image files.
+
+A dataset with a supported structure and [file formats](./datasets-adding#file-formats) automatically has a Dataset Viewer on its dataset page on the Hub. Any additional information about your dataset - such as captioning or object detection - is automatically loaded as long as you include this information in a metadata file (`metadata.csv`/`metadata.jsonl`).
+
+## Only images
+
+If your dataset only consists of one column with images, you can use a simple setup: store your image files at the root:
+
+```
+1.jpg
+2.jpg
+3.jpg
+4.jpg
+```
+
+or using subdirectories:
+
+```
+images/1.jpg
+images/2.jpg
+images/3.jpg
+images/4.jpg
+```
+
+Multiple [formats](./datasets-adding#file-formats) are supported at the same time, including png, jpg, tiff and webp.
+
+```
+images/1.jpg
+images/2.png
+images/3.tiff
+images/4.webp
+```
+
+## Additional columns
+
+If there is additional information you'd like to include about your dataset, like text captions or bounding boxes, add it as a `metadata.csv` file in your repository. This lets you quickly create datasets for different computer vision tasks like text captioning or object detection.
+
+```
+train/1.jpg
+train/2.jpg
+train/3.jpg
+train/4.jpg
+train/metadata.csv
+```
+
+Your `metadata.csv` file must have a `file_name` column which links image files with their metadata:
+
+```csv
+file_name,text
+1.jpg,a drawing of a green pokemon with red eyes
+2.jpg,a green and yellow toy with a red nose
+3.jpg,a red and white ball with an angry look on its face
+4.jpg,a cartoon ball with a smile on it's face
+```
+
+You can also use a [JSONL](https://jsonlines.readthedocs.io/en/latest/) file `metadata.jsonl`:
+
+```jsonl
+{"file_name": "1.jpg","text": "a drawing of a green pokemon with red eyes"}
+{"file_name": "2.jpg","text": "a green and yellow toy with a red nose"}
+{"file_name": "3.jpg","text": "a red and white ball with an angry look on its face"}
+{"file_name": "4.jpg","text": "a cartoon ball with a smile on it's face"}
+```
+
+Note that the image files and the metadata file for each split must reside in the same directory. The split name (e.g. `train` or `test`) will be taken from the directory name, see [File names and splits](./datasets-file-names-and-splits) for more information.
```

Comment:
not sure if the first phrase is needed. below it says that it's possible to use relative path, so maybe join this section and the one below and say from the beginning that `file_name` must be a full relative path? this way it doesn't really matter where you put it (am i right, @mariosasko ?)